include/backend/drm/bo_handle_table.h

```diff
@@ -0,0 +1,21 @@
+#ifndef BACKEND_DRM_BO_HANDLE_TABLE_H
+#define BACKEND_DRM_BO_HANDLE_TABLE_H
+
+/**
+ * Table performing reference counting for buffer object handles.
```

Comment:
Would be worth mentionning the [inspiration](https://gitlab.freedesktop.org/mesa/drm/-/blob/main/amdgpu/handle_table.c)

Reply:
Ack, added a link